resources/devices.json

```diff
@@ -30759,7 +30759,7 @@
       }
     },
     "Xiaomi Mi Pad": {
-      "standard": false,
+      "standard": true,
```

Comment:
All devices that are changed to `standard` here - should they really be? I've only heard of Google Pixel, but that's not to say they shouldn't be in `standard` - just checking they are "useful" enough in general?